index.js

```diff
@@ -43,3 +43,13 @@ async function isEnabled(options, { schema, source, context }) {
     return false
   }
 }
+
+export function explainGraphiQLPlugin() {
+  return {
+    name: 'mercurius-explain-graphiql-plugin',
```

Comment:
This is not a valid name. It should be used the same used in the `rollup` export. (`mercuriusExplain`)


```
      {
        file: packageJson.umd,
        format: 'umd',
        sourcemap: true,
        name: 'mercuriusExplain'
      }
```